terraform/modules/pollingAgent/main.tf

```
@@ -0,0 +1,74 @@
+locals {
+  test-cmd = <<-EOF
+    CMD bash -c exec 3<>"/dev/tcp/localhost/1080"
+    echo -en "GET /liveness HTTP/1.1\r\nHost: localhost:1080\r\nConnection: close\r\n\r\n">&3 &
+    grep Healthy <&3 &>/dev/null || exit 1
+    EOF
```

Better to just put the script here, and add the boilerplate in the healthcheck block (cf: next comment)

```suggestion
  test_cmd = <<-EOF
    exec 3<>"/dev/tcp/localhost/1080"
    echo -en "GET /liveness HTTP/1.1\r\nHost: localhost:1080\r\nConnection: close\r\n\r\n">&3 &
    grep Healthy <&3 &>/dev/null || exit 1
    EOF
```